pyvista/core/filters/rectilinear_grid.py

```diff
@@ -107,4 +115,17 @@ def to_tetrahedra(
 
         alg.SetInputData(self)
         _update_alg(alg, progress_bar, 'Converting to tetrahedra')
-        return _get_output(alg)
+        out = _get_output(alg)
+        if pass_cell_data:
+            # algorithm stores original cell ids in active scalars
+            for name in self.cell_data:  # type: ignore
+                if name != out.cell_data.active_scalars_name:
+                    out[name] = self.cell_data[name][out.cell_data.active_scalars]  # type: ignore
+
+        if alg.GetRememberVoxelId():
+            # original cell_ids are not named
+            if 'Unnamed_0' in out.cell_data:
+                # out.cell_data['vtkOriginalCellIds'] = out.cell_data.pop('Unnamed_0')
```

Comment:
Is this line intential, i.e. to describe what happens below, or was it left in from an attempted implementation?

Reply:
```suggestion
```